.github/workflows/cupertino.yml

```diff
@@ -36,7 +36,14 @@ jobs:
         name: Install dependencies
         run: flutter pub get
       - name: Check formatting
-        run: dart format --output=none --set-exit-if-changed .
+        # Don't lint the generated file native_cupertino_bindings.dart
+        # This approach is simpler than using `find` and excluding that file
+        # because `dart format` also excludes other file e.g. ones in
+        # directories start with '.'.
+        run: >
+          mv lib/src/native_cupertino_bindings.dart lib/src/native_cupertino_bindings.tmp &&
+          dart format --output=none --set-exit-if-changed . &&
```

Comment:
The `&&` means the file will not move back if any other files are not formatted correctly. That might not matter on CI though since we typically wouldn't run the other steps if the formatting check failed? I think leaving the mutated state on CI shouldn't hurt, but it makes me a little nervous.

Reply:
I'm not a bash expert but I think that this becomes significantly more complex because you have to save the exist code of `dart format` and then use it as the return value of the line.

Reply:
Yeah, I'd probably drop it in a subshell and use `trap` if we wanted to avoid the inconsistent state on failure.

```yaml
        run: >
          (set -e;
          mv lib/src/native_cupertino_bindings.dart lib/src/native_cupertino_bindings.tmp;
          cleanup() {
            mv lib/src/native_cupertino_bindings.tmp lib/src/native_cupertino_bindings.dart;
          };
          trap cleanup EXIT
          dart format --output=none --set-exit-if-changed .;)
```

I'd be fine either way  though - I don't think we currently care about repo consistency after a formatting failure so it's fine to keep the code simpler and focused on the happy case.